dojo/finding/urls.py

```diff
@@ -3,56 +3,125 @@
 from dojo.finding import views
 
 urlpatterns = [
+    # CRUD operations
+    re_path(
+        r'^finding/(?P<finding_id>\d+)$',
+        views.ViewFinding.as_view(),
+        name='view_finding'
+    ),
+    re_path(
+        r'^finding/(?P<finding_id>\d+)/edit$',
+        views.EditFinding.as_view(),
+        name='edit_finding'
+    ),
+    re_path(
+        r'^finding/(?P<finding_id>\d+)/delete$',
+        views.DeleteFinding.as_view(),
+        name='delete_finding'
+    ),
+    # Listing operations
+    re_path(
+        r'^finding$',
+        views.ListFindings.as_view(),
+        name='all_findings'
+    ),
+    re_path(
+        r'^finding/open$',
+        views.ListOpenFindings.as_view(),
+        name='open_findings'
+    ),
+    re_path(
+        r'^finding/verified$',
+        views.ListVerifiedFindings.as_view(),
+        name='verified_findings'
+    ),
+    re_path(
+        r'^finding/closed$',
+        views.ListClosedFindings.as_view(),
+        name='closed_findings'
+    ),
+    re_path(
+        r'^finding/accepted',
+        views.ListAcceptedFindings.as_view(),
+        name='accepted_findings'
+    ),
+    re_path(
+        r'^product/(?P<product_id>\d+)/finding/open$',
+        views.ListOpenFindings.as_view(),
+        name='product_open_findings'
+    ),
+    re_path(
+        r'^product/(?P<product_id>\d+)/findings$',
+        views.ListOpenFindings.as_view(),
+        name='view_product_findings_old'
```

Comment:
Should this be `views.ListFindings` to match `/finding/`? Also, what makes this one `_old`?